itou/www/apply/forms.py

```diff
@@ -534,7 +518,8 @@ def __init__(self, *args, siae, **kwargs):
                     "inverted_vae_contract",
                 )
             else:
-                self.fields.pop(geiq_field_name)
+                if geiq_field_name not in self.SIAE_OPTIONAL_FIELDS:
+                    self.fields.pop(geiq_field_name)
```

Comment:
![image](https://github.com/betagouv/itou/assets/88618/f0613255-4272-402e-8626-83914813c698)


Un jour je serai vraiment heureux quand on pourra découpler la logique GEIQ de la logique SIAE, en partie avec des formulaires spécifiques. Là, itérer sur des champs GEIQ et si jamais on est pas GEIQ alors on retire des champs mais seulement si ils font pas partie des champs optionnels des SIAE :exploding_head:

Reply:
C'est une idée de faire deux formulaires différents ("there is no lesser evil"), 
mais peut-être dans un deuxième temps et/ou au moment d'une refonte claire des structures.